contracts/operatorfilter/ERC1155OperatorFilterer.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/// @author: manifold.xyz
+
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "./CreatorOperatorFilterer.sol";
+import "../creator/extensions/ERC1155/ERC1155CreatorExtensionApproveTransfer.sol";
+
+contract ERC1155OperatorFilterer is ERC1155CreatorExtensionApproveTransfer, CreatorOperatorFilterer {
+    constructor(address operatorFilterRegistry, address subscriptionOrRegistrantToCopy, bool subscribe)
+        CreatorOperatorFilterer(operatorFilterRegistry, subscriptionOrRegistrantToCopy, subscribe)
+    {}
+
+    function supportsInterface(bytes4 interfaceId)
+        public
+        view
+        virtual
+        override (ERC1155CreatorExtensionApproveTransfer, AdminControl)
+        returns (bool)
+    {
+        return super.supportsInterface(interfaceId);
+    }
+
+    function approveTransfer(address operator, address from, address, uint256[] calldata, uint256[] calldata)
+        external
+        view
+        override
+        onlyAllowedOperator(operator, from)
+        returns (bool)
+    {
+        return true;
+    }
+
+    function mintNew(address creator, address[] calldata to, uint256[] calldata amounts, string[] calldata uris)
```

Comment:
option to just remove `creator` and accept in constructor. Accepting here enables a possible rare case where one creator can manage all their operator filters from one extension.